nav/process.py

```
@@ -17,21 +16,29 @@
+
+    image_file = image_files.image_files[0]
+    image_path = image_file.image_file_path
+
     with logger.open(str(image_path)):
         try:
-            s = inst_class.from_file(image_path)
+            snapshot = inst_class.from_file(image_path)
         except OSError as e:
             if 'SPICE(CKINSUFFDATA)' in str(e) or 'SPICE(SPKINSUFFDATA)' in str(e):
-                logger.error(f'No SPICE kernel available for "{image_path}"')
+                logger.error('No SPICE kernel available for "%s"', image_path)
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)